File: sample/src/main/java/me/kaelaela/sample/custom/jspoon/JSpoonOGData.java
package me.kaelaela.sample.custom.jspoon;

import pl.droidsonroids.jspoon.annotation.Selector;

public class JSpoonOGData {
    @Selector("head > title")
    public String title;
    @Selector(value = "meta[name=\"description\"]", attr = "content")
    public String desc;
    @Selector(value = "meta[property='og:image']", attr = "content")
    public String image;
    @Selector(value = "meta[property='og:url']", attr = "content")
    public String url;
}

File: sample/src/main/java/me/kaelaela/sample/custom/jspoon/JSpoonOGData.kt
package me.kaelaela.sample.custom.jspoon

import pl.droidsonroids.jspoon.annotation.Selector

class JSpoonOGData {
    @Selector("head > title")
    var title: String? = null
    @Selector(value = "meta[name=\"description\"]", attr = "content")
    var desc: String? = null
    @Selector(value = "meta[property='og:image']", attr = "content")
    var image: String? = null
    @Selector(value = "meta[property='og:url']", attr = "content")
    var url: String? = null
}
